README.md

```diff
@@ -101,8 +101,8 @@ e.g.
   for: 10m
   labels:
     severity: page
-    org: my-alternative-org
-    repo: specific-service-repository
+    atg_org: my-alternative-org
```

Comment:
typo? probably, I think the original `org` was typo.

```suggestion
    atg_owner: my-alternative-org
```

Reply:
I missed that. Thanks.